server/form-pages/apply/move-on/placementDuration.ts

```diff
@@ -106,7 +106,7 @@ export default class PlacementDuration implements TasklistPage {
       if (placementDate && placementDate.startDateSameAsReleaseDate === 'yes') {
         const releaseDate = basicInformation['release-date']
 
-        if (!releaseDate) return undefined
+        if (!releaseDate || !releaseDate?.releaseDate) return undefined
```

Comment:
Do we want to add a test for this?

Reply:
Have added one now, I think this happening is very unlikely due to the validation but just being extra careful